man/tpm2_getekcertificate.1.md

```diff
@@ -2,52 +2,72 @@
 
 # NAME
 
-**tpm2_getekcertificate**(1) - Retrieve the Endorsement key Certificate for the
-TPM endorsement key from the TPM manufacturer's endorsement certificate hosting
-server.
+**tpm2_getekcertificate**(1) - Retrieve the Endorsement key Certificate.
 
 # SYNOPSIS
 
 **tpm2_getekcertificate** [*OPTIONS*] [*ARGUMENT*]
 
 # DESCRIPTION
 
-**tpm2_getekcertificate**(1) - Retrieve the Endorsement key Certificate for
-the TPM endorsement key from the TPM manufacturer's endorsement certificate
-hosting server. The argument specifies the URL address for the ek certificate
-portal.
+**tpm2_getekcertificate**(1) - Retrieve the endorsement key certificate. The
+certificate is present either on the TCG specified TPM NV indices OR on the TPM
+manufacturer's endorsement certificate hosting server. The certificate lookup
+is in the following order:
+1. On the TPM NV indices: if an EK public file is **not specified** and argument
+   specifying the URL address for the EK certificate portal is **not supplied**.
+2. On the Intel EK certificate backend: if an EK public file, either RSA or ECC,
+   is **specified** and the tpmGeneratedEPS bit is clear and TPM manufacturer is
+   INTC.
+3. Provides information on how to download EK certificates from alternative
+   Intel EK certificate backend: if an EK public file, either RSA or ECC, is
+   **specified** and the tpmGeneratedEPS bit is set and TPM manufacturer is INTC.
+4. On a generic web hosting of EK certificates if an EK public, either RSA or
+   ECC, is **specified** and the argument specifying the URL address for the EK
+   certificate portal is **supplied**.
+
+Note: The default web address if not specified points to the Intel web hosting.
 
 # OPTIONS
 
   * **-o**, **\--ek-certificate**=_FILE_ or _STDOUT_:
 
-    The fileto save the Endorsement key certificate retrieved from the TPM
-    manufacturer provisioning server. Defaults to stdout if not specified.
+    The file to save the Endorsement key certificate. When EK certificates are
+    found on the TPM NV indices, this option can be specified additional times
+    to save the RSA and ECC EK certificates in order. The tool will warn if
+    additional EK certificates are found on the TPM NV indices and only a single
+    output file is specified. If the option isn't specified all the EK
+    certificates retrieved either from the manufacturer web hosting or from the
+    TPM NV indices, are output to stdout.
```

Comment:
Are they DER or PEM encoded? Perhaps you could just PEM encode them and spit them out N in a file and let the scissor lines delimit them. Might be easier.